src/main/kotlin/no/nav/syfo/infrastructure/clients/dokarkiv/dto/Dokumentvariant.kt

```diff
@@ -0,0 +1,63 @@
+package no.nav.syfo.infrastructure.clients.dokarkiv.dto
+
+enum class FiltypeType(
```

Comment:
Trenger man `…Type` på slutten her?
```suggestion
enum class Filtype(
```

Reply:
Nei, sikkert ikke 👍🏼